ServerCore/Pages/Pieces/InternMeta.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/InternMeta/{puzzleId}"
+@model ServerCore.Pages.Pieces.InternMetaModel
+
+@{
+    ViewData["Title"] = "Intern Puzzleday Meta";
```

Comment:
As mentioned below, I don't think this has to be limited to intern puzzleday.

Reply:
Not knowing what people want to do in nonintern events I do not want to make a page that can be used to possibly see the innards of those events.

Reply:
Thanks for the discussion and the suggestions. This page is now opt-in and has been extended to handle module metas as well.